cleverpush/src/main/java/com/cleverpush/stories/listener/OnSwipeTouchListener.java

```diff
@@ -0,0 +1,22 @@
+package com.cleverpush.stories.listener;
+
+import android.content.Context;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnTouchListener;
+
+public class OnSwipeTouchListener implements OnTouchListener {
+
+    public final GestureDetector gestureDetector;
```

Comment:
why is this made accessible to every other class?

if this is intended, please make this private and create a public getter returning this.gestureDetector